.do/deploy.template.yaml

```diff
@@ -0,0 +1,9 @@
+spec:
+  name: docker-stellar-core-horizon
```

Comment:
❔ Where does this name end up showing up? Just for my own knowledge.

Reply:
good question and I'm actually not sure. Says in the [docs](https://docs.digitalocean.com/products/app-platform/references/app-specification-reference/) it's the name of the app, but when you launch the app you can name it whatever you want. It may be a superfluous field